velox/substrait/tests/Substrait2VeloxPlanConversionTest.cpp

```diff
@@ -376,3 +468,43 @@ TEST_F(Substrait2VeloxPlanConversionTest, queryTest) {
     ASSERT_EQ(res, "{13613.1921}");
   }
 }
+
+// This test will firstly generate mock TPC-H lineitem ORC file. Then, Velox's
+// computing will be tested based on the generated ORC file.
+// Input: Json file of the Substrait plan for the first stage of the below
+// modified TPC-H Q1 query:
+//
+// select l_returnflag, l_linestatus, sum(l_quantity) as sum_qty,
+// sum(l_extendedprice) as sum_base_price, sum(l_extendedprice * (1 -
+// l_discount)) as sum_disc_price, sum(l_extendedprice * (1 - l_discount) * (1 +
+// l_tax)) as sum_charge, avg(l_quantity) as avg_qty, avg(l_extendedprice) as
+// avg_price, avg(l_discount) as avg_disc, count(*) as count_order from lineitem
+// where l_shipdate <= 10471 group by l_returnflag, l_linestatus order by
+// l_returnflag, l_linestatus
+//
+//  Tested Velox computings include: TableScan (Filter Pushdown) + Project +
+//  Aggregate
+//  Output: the Velox computed Aggregation result
+
+TEST_F(Substrait2VeloxPlanConversionTest, q1FirstStage) {
+  auto veloxConverter = std::make_shared<VeloxConverter>();
+  genLineitemORC(veloxConverter);
+  // Find and deserialize Substrait plan json file.
+  std::string planPath =
+      getDataFilePath("velox/substrait/tests", "data/q1_first_stage.json");
+  auto resIter = veloxConverter->getResIter(planPath);
```

Comment:
Check out https://github.com/facebookincubator/velox/pull/1686 for simplification of this code. Let's incorporate these changes.